.github/workflows/ci_summary.yaml

```diff
@@ -13,7 +13,7 @@ jobs:
 
     - name: "Build message"
       env:
-        SLACK_CHANNEL: "CEKV5LMK3"
+        SLACK_CHANNEL: "C03LGU37WU8"
```

Comment:
I think it might be a good idea to keep the existing line there, but commented out, and add the channel name as a comment so that we can easily switch between channels. Approving anyway since I do not think this is a blocker issue.